metadata/metadata_test.go

```diff
@@ -83,15 +83,15 @@ func TestValidateMetadata(t *testing.T) {
 		err error
 	}{
 		{Metadata{}, ErrValidatingData},
-		{Metadata{make(map[string]interface{})}, ErrValidatingData},
-		{Metadata{map[string]interface{}{}}, ErrValidatingData},
-		{Metadata{map[string]interface{}{"": nil}}, ErrValidatingData},
-		{Metadata{map[string]interface{}{"key": nil}}, ErrValidatingData},
-		{Metadata{map[string]interface{}{"key": "val"}}, ErrValidatingData},
-		{Metadata{map[string]interface{}{"DeviceType": "type"}}, ErrValidatingData},
-		{Metadata{map[string]interface{}{"DeviceType": nil, "ImageID": "image"}}, ErrValidatingData},
-		{Metadata{map[string]interface{}{"DeviceType": "device", "ImageID": "image"}}, nil},
-		{Metadata{map[string]interface{}{"DeviceType": "device", "ImageID": "image", "Data": "data"}}, nil},
+		{make(map[string]interface{}), ErrValidatingData},
+		{map[string]interface{}{}, ErrValidatingData},
```

Comment:
There's an alias `type Metadata map[string]interface{}` above, why not use it here?